tasty-plutus/src/Test/Tasty/Plutus/TestData.hs

```diff
@@ -69,25 +69,25 @@ data TestData (p :: Purpose) where
 {- | Describes whether a case is good (i.e. should pass) or bad (i.e. should
```

Comment:
The documentation should be adjusted to take into account `Outcome`'s new unified role.

Reply:
For the public API, the `Outcome` type is still only used for QuickCheck-based tests. So I don't think we'll change this part?

Reply:
I still tried to change the comments; overall I'm not sure if I should mention internal usages in user-facing docs.